_posts/2021-11-18-keras-updates/keras-updates.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "Keras for R is back!"
+description: > 
+  For a while, it may have seemed that Keras for R were in some undecidable state, like Schrödinger's cat before inspection. It is high time to  correct that impression. Keras for R is back, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks. 
+  This post provides a high-level overview. Future posts will go into more detail on some of the most helpful new features, as well as dive into the powerful low-level enhancements that make the former possible.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+  - name: Tomasz Kalinowski
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanakalinowskikeras
+date: 11-18-2021
+categories:
+  - Packages/Releases
+  - TensorFlow/Keras
+  - R
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Before we even talk about new features, let us answer the obvious question. Yes, there will be a second edition of *Deep Learning for R*! Reflecting what has been going on in the meantime, the new edition covers an extended set of proven architectures; at the same time, you'll find that intermediate-to-advanced designs already present in the first edition have become rather more intuitive to implement, thanks to the new low-level enhancements alluded to in the summary.
+
+But don't get us wrong -- the scope of the book is completely unchanged. It still is the perfect choice for people new to machine learning and deep learning. Starting from the basic ideas, it systematically progresses to intermediate and advanced topics, leaving you with both conceptual understanding and a bag of useful application templates.
```

Comment:
```suggestion
But don't get us wrong -- the scope of the book is completely unchanged. It is still the perfect choice for people new to machine learning and deep learning. Starting from the basic ideas, it systematically progresses to intermediate and advanced topics, leaving you with both a conceptual understanding and a bag of useful application templates.
```

Reply:
IDK why the still makes more sense to me where I've moved it. The "a" is just for list agreement.

Reply:
for (1) ... hm ... i had that syntax first :-(  bad luck ;-)

re (2) yeah, totally!